src/gigs-board/pages/community/Sidebar.jsx

```diff
@@ -0,0 +1,366 @@
+function href(widgetName, linkProps) {
+  linkProps = { ...linkProps };
+
+  if (props.nearDevGovGigsContractAccountId) {
+    linkProps.nearDevGovGigsContractAccountId =
+      props.nearDevGovGigsContractAccountId;
+  }
+
+  if (props.nearDevGovGigsWidgetsAccountId) {
+    linkProps.nearDevGovGigsWidgetsAccountId =
+      props.nearDevGovGigsWidgetsAccountId;
+  }
+
+  if (props.referral) {
+    linkProps.referral = props.referral;
+  }
+
+  const linkPropsQuery = Object.entries(linkProps)
+    .filter(([_key, nullable]) => (nullable ?? null) !== null)
+    .map(([key, value]) => `${key}=${value}`)
+    .join("&");
+
+  return `/#/${nearDevGovGigsWidgetsAccountId}/widget/gigs-board.pages.${widgetName}${
+    linkPropsQuery ? "?" : ""
+  }${linkPropsQuery}`;
+}
+/* END_INCLUDE: "common.jsx" */
+
+/* INCLUDE: "communities.jsx" */
+const communities = {
+  "zero-knowledge": {
+    overviewId: 397,
+    eventsId: 401,
+    icon: "https://ipfs.near.social/ipfs/bafkreiajwq6ep3n7veddozji2djv5vviyisabhycbweslvpwhsoyuzcwi4",
+    cover:
+      "https://ipfs.near.social/ipfs/bafkreihgxg5kwts2juldaeasveyuddkm6tcabmrat2aaq5u6uyljtyt7lu",
+    title: "Zero Knowledge",
+    desc: "Building a zero knowledge ecosystem on NEAR.",
+  },
+  protocol: {
+    overviewId: 412,
+    eventsId: 413,
+    icon: "https://ipfs.near.social/ipfs/bafkreidpitdafcnhkp4uyomacypdgqvxr35jtfnbxa5s6crby7qjk2nv5a",
+    cover:
+      "https://ipfs.near.social/ipfs/bafkreicg4svzfz5nvllomsahndgm7u62za4sib4mmbygxzhpcl4htqwr4a",
+    title: "Protocol",
+    desc: "Supporting the ongoing innovation of the NEAR Protocol.",
+  },
+  tooling: {
+    overviewId: 416,
+    eventsId: 417,
+    icon: "https://ipfs.near.social/ipfs/bafkreie2eaj5czmpfe6pe53kojzcspgozebdsonffwvbxtpuipnwahybvi",
+    cover:
+      "https://ipfs.near.social/ipfs/bafkreiehzr7z2fhoqqmkt3z667wubccbch6sqtsnvd6msodyzpnf72cszy",
+    title: "Tooling",
+    desc: "Supporting the ongoing innovation of tooling.",
+  },
+  "contract-standards": {
+    overviewId: 414,
+    eventsId: 415,
+    icon: "https://ipfs.near.social/ipfs/bafkreiepgdnu7soc6xgbyd4adicbf3eyxiiwqawn6tguaix6aklfpir634",
+    cover:
+      "https://ipfs.near.social/ipfs/bafkreiaowjqxds24fwcliyriintjd4ucciprii2rdxjmxgi7f5dmzuscey",
+    title: "Contract Standards",
+    desc: "Coordinating the contribution to the NEAR dapp standards.",
+  },
+};
+/* END_INCLUDE: "communities.jsx" */
+
+/* INCLUDE: "mockcommunity.jsx" */
+const socialLinks = [
+  {
+    href: "#/mob.near/widget/ProfilePage?accountId=self.social.near",
+    iconClass: "bi bi-globe",
+    label: community.title,
+  },
+
+  {
+    href: "https://github.com/NearSocial",
+    iconClass: "bi bi-github",
+    label: community.title,
+  },
+  {
+    href: "https://twitter.com/NearSocial_",
+    iconClass: "bi bi-twitter",
+    label: community.title,
+  },
+  {
+    href: "https://t.me/NearSocial",
+    iconClass: "bi bi-telegram",
+    label: community.title,
+  },
+];
+
+const mockTeamMembers = [
```

Comment:
communities and mockTeamMembers can be replaced by @near-akaia-root 's PR on loading community and community admins. Are you plan to make another PR after this one, and Carina's? But seems it's not a good idea to have mock team member in production so I would suggest push Carina's PR first and update this PR to use real data

Reply:
updated and will make a new pr to Carina's branch